plugins/ahmadfatoni/apigenerator/controllers/api/UsersAPIController.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace AhmadFatoni\ApiGenerator\Controllers\API;
+
+use Cms\Classes\Controller;
+
+use Illuminate\Http\Request;
+use AhmadFatoni\ApiGenerator\Helpers\Helpers;
+use Tymon\JWTAuth\Facades\JWTAuth;
+use RainLab\User\Models\User;
+
+class UsersAPIController extends Controller
+{
+    protected $User;
```

Comment:
Chỗ này tôi nghĩ viết thường thôi ông à $user
```suggestion
    protected $user;
```

Reply:
Cái này là do cái plugin API tự nó sinh ra :)) đổi thì cũng dc nmà sau dùng cái đó sinh ra mình lại đổi tiếp chứ?